webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -117,13 +138,39 @@ export const ErrataTab = () => {
     </Split>
   );
 
+  let toggleGroup;
+  if (!contentFacet.contentViewDefault && !contentFacet.lifecycleEnvironmentLibrary) {
+    toggleGroup = (
+      <ToggleGroup aria-label="Installable Errata">
+        <ToggleGroupItem
+          text={__('All')}
+          buttonId="allToggle"
+          aria-label="Show All"
+          isSelected={toggleGroupState === all}
+          onChange={() => setToggleGroupState(all)}
+        />
+
+        <ToggleGroupItem
+          text={__('Installable')}
+          buttonId="installableToggle"
+          aria-label="Show Installable"
+          isSelected={toggleGroupState === installable}
+          onChange={() => setToggleGroupState(installable)}
+        />
+      </ToggleGroup>
+    );
+  }
+
   return (
     <div>
       <div id="errata-alert">
+        { (toggleGroupState === all) &&
         <Alert variant="info" isInline title={__('Note')}>
-          {__('Errata management functionality on this page is incomplete.')} {' '}
-          <a href={urlBuilder(`content_hosts/${hostId}/errata`, '')}>{__('Visit the previous Errata page') }.</a>
+          {__(" Some of the Errata shown below may not be installable as they are not in this Host's Content View and Lifecycle Environment.")}
+          {__('In order to apply such Errata an Incremental Update is required. ')}
+          <a href={urlBuilder('errata', '')}>{__('Select Errata for an Incremental Update.') }.</a>
         </Alert>
+         }
```

Comment:
I still think this should be removed

Reply:
But dont they need to know incremental update is required ?

Reply:
I basically took the text from the previous version. If its not adding any value I am ok removing it.

Reply:
I don't see any value, but I'm also not well-versed in incremental updates, so I'd like to get feedback from others..

Reply:
For now going to remove this line and address it in a future PR